docs/source/index.rst

```diff
@@ -38,7 +45,10 @@ complete documentation for using Cartographer with ROS at the
 .. _Cartographer ROS Read the Docs site: https://google-cartographer-ros.readthedocs.io
 
 Getting started without ROS
-===========================
+---------------------------
+
+Please see our ROS integration as a starting point for integrating your system
+with the standalone library. Currently, it is the best available reference.
```

Comment:
mention that it is kept up to date with development through CI and is therefore always correct.